atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -63,31 +66,61 @@ public SchemaMutationLockTables(CassandraClientPool clientPool, CassandraKeyValu
     }
 
     public TableReference createLockTable() throws TException {
-        return clientPool.runWithRetry(this::createLockTable);
-    }
-
-    private TableReference createLockTable(CassandraClient client) throws TException {
-        String lockTableName = LOCK_TABLE_PREFIX + "_" + getUniqueSuffix();
-        TableReference lockTable = TableReference.createWithEmptyNamespace(lockTableName);
+        TableReference lockTable = TableReference.createWithEmptyNamespace(LOCK_TABLE_PREFIX);
```

Comment:
Do we still need to have a suffix in the table name for the other schema mutation lock code work?